packages/route-viewer-overlay/src/RouteViewerOverlay.story.js

```diff
@@ -32,3 +36,28 @@ export const WithPathStyling = () => (
     />
   </BaseMap>
 );
+
+// Storyshot can't render a story with hooks. Creating a function like this
+// is the workaround. See https://github.com/storybookjs/storybook/issues/8177#issuecomment-599866282
+const FlexRouteWithClipButton = () => {
+  const [clip, setClip] = useState(true);
+  return (
+    <>
+      <button type="button" onClick={() => setClip(!clip)}>
+        {clip ? "Unclip" : "Clip"} Route to Outside Flex Zone
+      </button>
```

Comment:
Replace this button with "story args", see https://storybook.js.org/blog/introducing-storybook-args/ and examples in #323 or #281.

Reply:
I've moved to args and added a control in 6945aafe0033c070e7556d47894b2c7de63b4441, but worry that it's a bit less discoverable than the button. Moving to template also has required a few hacks